src/Application/Models/Asset.js

```diff
@@ -1,13 +1,15 @@
 import { assetDefaults } from './../../util/constants';
 
 class Asset {
-  constructor(name, price, multiplier, owned, unlocked) {
+  constructor(name, price, multiplier, owned, unlocked, investment, revenue) {
     this._name = name;
     this._price = price;
     this._multiplier = multiplier;
     this._owned = owned;
     this._unlocked = unlocked;
     this._defaults = assetDefaults[name];
+    this._investment = investment;
```

Comment:
nit: with no context, idk what `investment` means.  My first guess is this is total amount of money spent on Assets of this type, correct?